src/current/_includes/v23.1/sql/privileges.md

```diff
@@ -16,17 +16,17 @@ Privilege | Levels | Description
 `EXTERNALCONNECTION` | System | Grants the ability to connect to external systems such as object stores, key management systems, Kafka feeds, or external file systems. Often used in conjunction with the `BACKUP`, `RESTORE`, and `CHANGEFEED` privilege.
 `EXTERNALIOIMPLICITACCESS` | System | Grants the ability to interact with external resources that require implicit access.
 `INSERT` | Table, Sequence | Grants the ability to insert objects at the table or sequence level.
-`MODIFYCLUSTERSETTING` | System | Grants the ability to modify [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}).
+<a name="modifyclustersetting"></a>`MODIFYCLUSTERSETTING` | System | Grants the ability to modify [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}).
```

Comment:
is this link right? you're not adding anything inside the link component, so doesn't seems like there will be something to click on.
Same for all cases below

Reply:
Yes, those manual anchors are correct, see the [docs style guide](https://github.com/cockroachdb/docs/blob/main/StyleGuide.md#links-to-a-specific-location-on-a-page-that-is-not-a-heading) for more info.

Reply:
The docs shows `<a id="flags-max-offset"></a>` and you used `name` instead of `id`, does it also work?

Reply:
Yes, the `name` attribute works as well as `id`.